tests/unit/RelationDAOTest.php

```diff
@@ -1,73 +1,81 @@
 <?php
 
-class RelationDAOTest extends CDbTestCase
+/**
+ * run with:
+ * docker-compose run --rm test ./vendor/codeception/codeception/codecept run unit RelationDAOTest
```

Comment:
**suggestion**: I think it's worth adding this into the `How to test?` section of the Pull Request summary.
**issue**: The reason for the above suggestion is that RelationDAOTest is failing when run in my `dev` environment because of `[PHPUnit_Framework_Exception] Trying to get property 'id' of non-object` error. However, the unit test is passing in my CI pipeline. Feels like unit tests on dev environment is using test data from `data/dev/` directory whilst unit tests on CI pipeline are being run against data from `data/production_like/`.
**suggestion**: DOI 100039 (id=5) is a DOI found in both dev and production_like data so might enable RelationDAOTest to pass in both environments.